octue/cloud/pub_sub/service.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, backend, service_id=None, run_function=None, *args, **kwargs)
 
         self.backend = backend
         self.run_function = run_function
-        self._credentials = GCPCredentialsManager(backend.credentials_environment_variable).get_credentials()
```

Comment:
@thclark I'm unsure whether we need `backend.credentials_environment_variable` or not anymore

Reply:
I think it should be removed TBH, technically a breaking change but see my comments on the PR

Reply:
After discussion, we've decided to only use `GOOGLE_APPLICATION_CREDENTIALS`  as the environment variable name, removing the need for this